ui/tests/acceptance/oidc-provider-test.js

```diff
@@ -66,7 +66,7 @@ const entityAlias = async function (entityId, accessor, groupId) {
   return consoleComponent.lastLogOutput.includes('Success');
 };
 const setupWebapp = async function (redirect) {
-  let webappName = `my-webapp-${new Date().getTime()}`;
```

Comment:
on multiple reruns of tests - this caused failures in the `/oidc-config` because it generates multiple providers and apps tests so changed to a consistent name, and cleaned up the app at the end of each test

Reply:
Smart. i've run into issues with using any kind of date in the name for testing. It's also easier to debug when using something consistent.